tests/document/instance.py

```diff
@@ -1,33 +1,30 @@
 # -*- coding: utf-8 -*-
-import bson
 import os
 import pickle
 import unittest
 import uuid
-import warnings
 import weakref
 from datetime import datetime
 
+import bson
 from bson import DBRef, ObjectId
 from pymongo.errors import DuplicateKeyError
 from six import iteritems
 
-from mongoengine.mongodb_support import get_mongodb_version, MONGODB_36, MONGODB_34
```

Comment:
Minor tip: It's generally good to decouple logical changes from trivial stuff like formatting or fixing an order of imports (which ideally would be maintained automatically via linting tools). Mashing these together in a single commit makes it harder to review.